markdown/extensions/tables.py

```diff
@@ -85,7 +90,60 @@ def _split_row(self, row, border):
                 row = row[1:]
             if row.endswith('|'):
                 row = row[:-1]
-        return row.split('|')
+        return self._split(row, '|')
+
+    def _split(self, row, marker):
+        """ split a row of text with some code into a list of cells. """
+        if self._row_has_unpaired_backticks(row):
+            # fallback on old behaviour
+            return row.split(marker)
+        backtick_pattern = BacktickPattern('^' + BACKTICK_RE) # modify the backtick pattern to only match at the beginning of the search string
+        escaped = False
+        elements = []
+        current = ''
+        i = 0
+        while i < len(row):
+            letter = row[i]
+            if letter == marker:
+                if current != '' or len(elements) == 0:
+                    # Don't append empty string unless it is the first element
+                    # The border is already removed when we get the row, then the line is strip()'d 
+                    # If the first element is a marker, then we have an empty first cell
+                    elements.append(current)
+                current = ''
+                escaped = False
+            else:
+                match = backtick_pattern.getCompiledRegExp().match(row[i:])
+                if not match:
+                    if escaped:
+                        if letter not in [' ', '\t', '\n']:
+                            # if we added a code element in this cell, add to that element's text
+                            elements[-1].text += letter
+                    else:
+                        current += letter
+                else:
+                    # match.groups()[-1] is the rest of the (unmatched) text
+                    # jump pointer to the beginning of the rest of the text
+                    i = row.index(match.groups()[-1])
+                    # build the code element
```

Comment:
Please don't do this. Lets keep all inline parsing in the inline parsing step of the parser. If you want to use the regex for codespans to identify them, I understand, but when the table blockparser is done, the cell should still contain Markdown formatted inline text. If you don't do it that way, then other (inline) extensions could fail in unpredictable ways on table cells with codespans.